style.css

```diff
@@ -0,0 +1,128 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    min-height: 100vh;
+    background-color: #d3d3d3;
+    font-family: 'Roboto', sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.container{
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    perspective: 2000px;
+    perspective-origin: 50% 50%;
+}
+.cover{
+    position: absolute;
+    min-width: 400px;
+    max-width: 800px;
+    height: 480px;
+    margin: 0 auto;
+    background-image: url(img/cover.jpg);
+    border-radius: 10px;
+    box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.5);
+    transform-origin: left;
+    animation: animate 6s ease-in-out normal forwards;
+    z-index: 1;
+}
+.cover::before{
+    content: '';
+    width: 40px;
+    background: radial-gradient(#575450 6px,transparent 7px) repeat-y;
+    background-size: 50px 30px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    box-sizing: border-box;
+}
+.cover::after{
+    content: '';
+    position: absolute;
+    top: 0;
+    left: -20px;
+    bottom: 0;
+    width: 60px;
+    background: linear-gradient(transparent,transparent 13px,black 14px,black 16px,transparent 17px,transparent 30px) repeat-y;
+    box-sizing: border-box;
```

Comment:
You've already reset to `box-sizing: border-box` why you use this twice ??? @murtazshaikh

Reply:
My mistake, I have removed it.